src/Microsoft.DotNet.PlatformAbstractions/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "description": "Abstractions for making code that uses file system and environment testable.",
-  "version": "1.1.4-servicing-*",
```

Comment:
We haven't been shipping PlatformAbstractions and DependencyModel so lets not bump there versions until we think we need to ship them (we may even want to consider downgrading them to one higher then the last one we shipped).

cc @eerhardt

Reply:
Ok. Will fix that then. So 1.1.4-servicing-* is good? Or should we downgrade to 1.1.3?

Reply:
Just revert to 1.1.4 for now.

Reply:
Ok

Reply:
This sounds like a good plan to me.

Note, for release/2.0.0, when we merge in [the build changes from master](https://github.com/dotnet/core-setup/pull/3235), these packages will automatically have the version number that the whole repo uses.  We won't have these versions spread all over the repo.

Of course, we will be able to override the version number for each package, but by default these numbers won't be duplicated.